.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,58 @@
+name: Bump versions for extension only
```

Comment:
Now that this exists in two .yml files, we should probably differentiate the name (by e.g. mentioning in brackets how it is triggered, for what branch etc)